site-scrapers/Harrington-acuity/config.js

```diff
@@ -0,0 +1,43 @@
+const site = {
+    name: "Harrington (local residents only)",
+    street: "153 Chestnut Street",
+    city: "Southbridge",
+    zip: "01550",
+    website:
+        "https://app.acuityscheduling.com/schedule.php?owner=22192301&calendarID=5202050",
+    restrictions:
+        "Registration is open to only those who live in the following towns: Auburn, Brimfield, Brookfield, (North, East, West) Charlton, Dudley, Holland, Leicester, Oxford, Southbridge, Spencer, Sturbridge, Sutton, Uxbridge, Wales, Warren, Webster",
+};
+
+const locations = [
+    {
+        name: "Harrington &emdash; local residents",
```

Comment:
is there a reason you're using `&emdash;` here? React escapes everything by default so I think it'll show up literally like that on the front end.

Reply:
I didn't know about React's behavior. I'll use the wording from the original Harrington scraper.

Reply:
Actually, I have a question about the wording: the original, "Southbridge Community Center", doesn't connect the site to Harrington Healthcare. To me, it gives the impression that the site is being run by a public health entity. Do we maintain that wording, or use "Harrington Healthcare (town-restricted)" and "Harrington Healthcare (not town-restricted)"? (Attempting to boil down the wording on their vaccination landing page, "For those who live outside of these towns", to a few words that make sense.)